src/NovelRT/Graphics/Vulkan/VulkanGraphicsContext.cpp

```diff
@@ -358,8 +358,9 @@ namespace NovelRT::Graphics::Vulkan
             std::static_pointer_cast<VulkanGraphicsBuffer>(vertexBufferRegion.GetCollection());
         VkBuffer vulkanVertexBuffer = vertexBuffer->GetVulkanBuffer();
         size_t vulkanVertexBufferOffset = vertexBufferRegion.GetOffset();
+        VkDeviceSize castedVulkanVertexBufferOffset = static_cast<VkDeviceSize>(vulkanVertexBufferOffset);
 
-        vkCmdBindVertexBuffers(vulkanCommandBuffer, 0, 1, &vulkanVertexBuffer, &vulkanVertexBufferOffset);
+        vkCmdBindVertexBuffers(vulkanCommandBuffer, 0, 1, &vulkanVertexBuffer, &castedVulkanVertexBufferOffset);
```

Comment:
`size_t` is different on macOS :^) If this isn't safe on other OSes, I can ifdef it.... I rather not though if this doesn't cause a failure on the others.